Packs/Base/Scripts/ValidateContent/ValidateContent.py

```diff
@@ -0,0 +1,384 @@
+
+import git
+from demisto_sdk.commands.common.constants import ENTITY_TYPE_TO_DIR
+from demisto_sdk.commands.common.content import Content
+from demisto_sdk.commands.common.content.objects.pack_objects.pack import Pack
+from demisto_sdk.commands.common.tools import find_type
+from demisto_sdk.commands.init.contribution_converter import (
+    AUTOMATION, INTEGRATION, INTEGRATIONS_DIR, SCRIPT, SCRIPTS_DIR,
+    ContributionConverter, format_manager, get_child_directories,
+    get_child_files)
+from demisto_sdk.commands.lint.lint_manager import LintManager
+from demisto_sdk.commands.split_yml.extractor import Extractor
+from demisto_sdk.commands.validate.validate_manager import ValidateManager
+from ruamel.yaml import YAML
+
+import demistomock as demisto
+from CommonServerPython import *
+
+CACHED_MODULES_DIR = '/tmp/cached_modules'
+yaml = YAML()
+
+
+def _create_pack_base_files(self):
+    """
+    Create empty 'README.md', '.secrets-ignore', and '.pack-ignore' files that are expected
+    to be in the base directory of a pack
+    """
+    fp = open(os.path.join(self.pack_dir_path, 'README.md'), 'a')
+    fp.close()
+
+    fp = open(os.path.join(self.pack_dir_path, '.secrets-ignore'), 'a')
+    fp.close()
+
+    fp = open(os.path.join(self.pack_dir_path, '.pack-ignore'), 'a')
+    fp.close()
+
+
+def content_item_to_package_format(
+        self, content_item_dir: str, del_unified: bool = True, source_mapping: Optional[Dict] = None  # noqa: F841
+) -> None:
+    child_files = get_child_files(content_item_dir)
+    for child_file in child_files:
+        cf_name_lower = os.path.basename(child_file).lower()
+        if cf_name_lower.startswith((SCRIPT, AUTOMATION, INTEGRATION)) and cf_name_lower.endswith('yml'):
+            content_item_file_path = child_file
+            file_type = find_type(content_item_file_path)
+            file_type = file_type.value if file_type else file_type
+            try:
+                extractor = Extractor(
+                    input=content_item_file_path, file_type=file_type, output=content_item_dir, no_logging=True,
+                    no_pipenv=True)
+                extractor.extract_to_package_format()
+            except Exception as e:
+                err_msg = f'Error occurred while trying to split the unified YAML "{content_item_file_path}" ' \
+                          f'into its component parts.\nError: "{e}"'
+                self.contrib_conversion_errs.append(err_msg)
+            if del_unified:
+                os.remove(content_item_file_path)
+
+
+def format_converted_pack(self) -> None:
+    """Runs the demisto-sdk's format command on the pack converted from the contribution zipfile"""
+    from_version = '6.0.0' if self.create_new else ''
+    format_manager(
+        input=self.pack_dir_path, from_version=from_version, no_validate=True, update_docker=True, assume_yes=True
+    )
+
+
+def convert_contribution_to_pack(contrib_converter: ContributionConverter) -> None:
```

Comment:
this function and all the above are copied from the SDK without the click logs.
can we use the original while disabling click functionality (or redirecting it to run `demisto.debug` instead)?

Reply:
its not just about the click logs, we also removed running format
unless you insist, i prefer to leave it like that for now